<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UILiveLessonWnd</class>
 <widget class="QWidget" name="UILiveLessonWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UILiveLessonWnd</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <widget class="QLabel" name="title_label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>70</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>选择课程</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>5</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>451</width>
        <height>150</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QScrollArea{border: 1px solid #ffffff;background-color: rgb(255, 255, 255);}</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>449</width>
         <height>148</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="back_label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="title_pushButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>481</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 217, 217);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>back_label</zorder>
     <zorder>title_pushButton</zorder>
     <zorder>title_label</zorder>
     <zorder>pushButton</zorder>
     <zorder>scrollArea</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
